keras/layers/attention/attention_test.py

```diff
@@ -99,3 +99,14 @@ def test_attention_errors(self):
 
         with self.assertRaisesRegex(ValueError, "length 2 or 3"):
             layer([tensor, tensor], mask=[tensor])
+
+    def test_attention_with_dropout(self):
+        layer = layers.Attention(dropout=0.2)
+        query = np.array([[[1.0, 0.0], [0.0, 1.0]]])
+        value = np.array([[[1.0, 1.0], [1.0, 1.0]]])
+        output, scores = layer(
+            [query, value],
+            return_attention_scores=True,
+        )
+        self.assertAllClose(output, [[[1.0, 1.0], [1.0, 1.0]]])
```

Comment:
I'm confused -- if there's dropout the test should be non deterministic, no? So the output should change from one execution to another?